01_data_exploration.ipynb

```diff
@@ -10,17 +10,17 @@
     "# 1. Data Exploration\n",
```

Comment:
<p>This cell causes an exception</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/informatics-lab/ml_weather_tutorial/pull/21/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='58'/>

Reply:
<p>at some point i changed variable names and for some reason this cell didn't get updated (not sure  how i ran the notebook previously). Fixed niow</p>